classes/publish/app.js

```diff
@@ -134,129 +142,161 @@ module.exports = class PublishApp {
-            this.log.warn(err.message);
-
-            this.log.debug('Cleaning up');
-            if (fs.existsSync(this.path)) {
-                rimraf.sync(this.path);
-            }
-
-            return false;
-        }
-
-        // create main css bundle
-        this.log.debug('Creating css bundle file');
-        try {
-            if (this.css) {
-                const input = join(this.cwd, this.css);
-                const precss = fs.readFileSync(input, 'utf8');
-                const processor = postcss(autoprefixer());
+                            babelrc: false,
+                        }),
+                        rollupReplace({
+                            'process.env.NODE_ENV': JSON.stringify(
+                                'production',
+                            ),
+                        }),
+                        terser(),
+                    ],
+                    input: join(this.cwd, this.js),
+                };
+                const bundled = await rollup.rollup(options);
+                await bundled.write({
+                    format: 'iife',
+                    file: join(this.path, 'ie11/index.js'),
+                    sourcemap: true,
+                });
+            } catch (err) {
+                this.log.error('Unable to create bundle file');
+                this.log.warn(err.message);
 
-                processor.use(cssnano());
+                this.log.debug('Cleaning up');
+                if (fs.existsSync(this.path)) {
+                    rimraf.sync(this.path);
+                }
 
-                const result = await processor.process(precss, {
-                    from: this.css.replace(/(.*\/)*/, ''),
-                    to: 'index.css',
-                    map: { inline: false },
-                });
-                fs.writeFileSync(join(this.path, 'main/index.css'), result.css);
-                fs.writeFileSync(
-                    join(this.path, 'main/index.css.map'),
-                    result.map,
-                );
-            } else {
-                this.log.debug('CSS assets not specified');
+                return false;
             }
-        } catch (err) {
-            this.log.error('Unable to create css bundle file');
-            this.log.warn(err.message);
+        } else {
+            this.log.debug(
+                'JavaScript entrypoint not defined, skipping JS bundling',
+            );
+        }
 
-            this.log.debug('Cleaning up');
-            if (fs.existsSync(this.path)) {
-                rimraf.sync(this.path);
+        if (this.css) {
+            // create main css bundle
+            this.log.debug('Creating css bundle file');
+            try {
+                if (this.css) {
+                    const input = join(this.cwd, this.css);
+                    const precss = fs.readFileSync(input, 'utf8');
+                    const processor = postcss(autoprefixer());
```

Comment:
What is postcss actually doing here?

Reply:
Do you mean autoprefixer?
It adds vendor prefixing: https://github.com/postcss/autoprefixer

Or do you mean what do we use it for in general?
If the later it:

* Inlines dependencies from @import statements
* minifies
* adds vendor prefixing
* creates source maps